tasks.py

```diff
@@ -16,7 +16,7 @@
     'no-value-for-parameter',
     'method-hidden'
 ])
-TEST_TARGETS = ' '.join([PACKAGE_NAME, 'tests'])
+TEST_TARGETS = ' '.join([PACKAGE_NAME, 'tests/test_*'])
```

Comment:
Can probably leave this change off once the `tests/_compat.py` is moved to a `test_*.py` file.